feat: use KinD for local kubernetes cluster (#6)
elmy-team/devops-interview: README.md

- Your manifests formats (helm, terraform, raw, ..)
- The scaling capabilities of your deployment

To do so, it's recommended to install a Kubernetes environment on your machine by using `KinD` with an `ingress-nginx service`. An installation memo is proposed at the end of this page.

### Bonus

We would like to make sure that the app is accessible using the ingress. So if you still have time:

> - The deployed ingress should only allow whitelisted IP addresses. You can pick an IP range of your choice. Note that the ingress controller deployed in the cluster is [Nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/).

Reviewer: ```suggestion
- The deployed ingress should only allow whitelisted IP addresses. You can pick an IP range of your choice. Note that the ingress controller that should be deployed in the cluster is [Nginx Ingress Controller](https://kubernetes.github.io/ingress-nginx/).
```

### Local kubernetes installation using Kind

> - Install KinD on your machine [this README file](https://github.com/oui-team/devops-interview/tree/master/app).
> - Create a local cluster with `Kubernetes` installed.
> - Install [Helm](https://helm.sh/fr/docs/).
> - Deploy the `Helm chart` on your KinD cluster by running:

Reviewer: ```suggestion
- [Install KinD on your machine](https://github.com/oui-team/devops-interview/tree/master/app).
- Create a local Kubernetes cluster using KinD.
- Install [Helm](https://helm.sh/fr/docs/).
- Deploy the `ingress-nginx` Helm chart on your KinD cluster by running:
```
